### PR TITLE
completions: update bash completions with recent command changes

### PR DESCRIPTION
**Pull request by grondo**
The bash completions are slightly out of date. This PR updates them (for the changes I found) and also fixes a minor annoyance I have noticed lately.

### CODE REVIEW

File: etc/completions/flux.pre [hunk 1]
```diff
@@ -1067,10 +1077,22 @@ _flux_restore()
...
     #  flux-top(1) can target jobids that are also instances
     local jobs=$(flux jobs -no {uri}:{id} | grep -v ^None: \
                  | sed -n 's/.*://p')
-    COMPREPLY=( $(compgen -W "$jobs" -- "$cur") )
+    COMPREPLY=( $(compgen -W "${OPTS} $jobs" -- "$cur") )
+    if [[ "${COMPREPLY[@]}" == *= ]]; then
+        # Add space if there is not a '=' in suggestions
+        compopt -o nospace
+    fi
+    return 0
```
[the remark this answers is not in the document]
Comment (grondo) [reply etc/completions/flux.pre:1095]: No, flux-top didn't support any options before and this chunk of code adds support for it.

[the remark this answers is not in the document]
Comment (chu11) [reply etc/completions/flux.pre:1095]: ahh ok i see

**Review by chu11 — APPROVED** [review]
LGTM!

**Comment by grondo** [comment]
Thanks! I've set MWP.

**Pull request merged**